package.json

```diff
@@ -4,9 +4,18 @@
   "description": "The back end of the warriorz",
   "main": "server/index.js",
   "scripts": {
-    "test": "export NODE_ENV=test && nyc --reporter=html --reporter=text mocha --require @babel/register --timeout 5000 ./server/tests/*.js --exit",
+    "test1": "set NODE_ENV=test && npm run drop:test  && npm run testTables && nyc --reporter=html --reporter=text mocha --require @babel/register --timeout 5000 ./server/tests/*.test.js --exit",
```

Comment:
Why do we have two test scripts? Clean up one

Reply:
I will remove it.